actions/graybox/pause-project-in-queue.js

```diff
@@ -0,0 +1,80 @@
+/* ************************************************************************
+* ADOBE CONFIDENTIAL
+* ___________________
+*
+* Copyright 2024 Adobe
+* All Rights Reserved.
+*
+* NOTICE: All information contained herein is, and remains
+* the property of Adobe and its suppliers, if any. The intellectual
+* and technical concepts contained herein are proprietary to Adobe
+* and its suppliers and are protected by all applicable intellectual
+* property laws, including trade secret and copyright laws.
+* Dissemination of this information or reproduction of this material
+* is strictly forbidden unless prior written permission is obtained
+* from Adobe.
+************************************************************************* */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const initFilesWrapper = require('./filesWrapper');
+const { getAioLogger } = require('../utils');
+/**
+ * This Action Sets the project status to paused in Project Queue & the Project Status JSON of that project
+ */
+async function main(params) {
+    const logger = getAioLogger();
+    const filesWrapper = await initFilesWrapper(logger);
+    let responsePayload = 'Graybox Pause Project in Project Queue action invoked';
+    let responseCode = 200;
+    logger.info(responsePayload);
+    try {
+        const { projectPath } = params;
+        logger.info(`Project to be paused :: ${projectPath}`);
+        if (await filesWrapper.fileExists('graybox_promote/project_queue.json')) {
```

Comment:
nit: this can be a var `graybox_promote/project_queue.json` as it is used in multiple places

Reply:
Updated, thanks!